requirements.txt

```diff
@@ -6,7 +6,7 @@
 #
 ansible-compat==25.6.0
     # via ansible-lint
-ansible-core==2.17.12
+ansible-core==2.17.13
```

Comment:
@devanshjainms did this version of ansible-core work with the earlier releases like SLES 15.4, and RHEL 8.8? While we can control the python version on the controller, as far as I know, the default python version on these OS versions is python 3.6. [ansible-core-support-matrix](https://docs.ansible.com/ansible/latest/reference_appendices/release_and_maintenance.html#ansible-core-support-matrix)

Reply:
these requirements are only used to create virtual env where the python unit tests are executed